tests/ext/test_application.py

```diff
@@ -210,6 +210,21 @@ def test_job_queue(self, one_time_bot, app, recwarn):
         assert str(recwarn[0].message) == expected_warning
         assert recwarn[0].filename == __file__, "wrong stacklevel"
 
+    def test_concurrent_updates(self, one_time_bot):
+        application = ApplicationBuilder().bot(one_time_bot).concurrent_updates(2).build()
+        assert application.concurrent_updates == (
+            application.update_processor.max_concurrent_updates
+        )
+
+    def test_active_updates(self, one_time_bot):
```

Comment:
it would be nice to mock process_update to check if number of active updates is actually correct